/ros2_tcp_bridge:
  ros__parameters:
    vSpeed: 80
    turnSpeed: 100
